bundles/org.openhab.binding.warmup/README.md

```diff
@@ -0,0 +1,68 @@
+
+Any Warmup 4iE device(s) must be registered at https://my.warmup.com/ prior to usage.
+
+This API is not known to be documented publicly. The binding api implementation has been derived from the implementations at https://github.com/alyc100/SmartThingsPublic/blob/master/devicetypes/alyc100/warmup-4ie.src/warmup-4ie.groovy and https://github.com/alex-0103/warmup4IE/blob/master/warmup4ie/warmup4ie.py. 
+
+## Supported Things
+
+* Bridge - Credentials to my.warmup.com which allow the rest of the system to function
+* Room - An individual Warmup 4iE device which is a WiFi connected device which controls a heating circuit. The device is optimised for controlling underfloor heating (electric or hydronic), although it can also control central heating circuits. The device reports the temperature from one of two thermostats, either a floor temperature probe or the air temperature at the device. The separate temperatures do not appear to be reported through the API. It appears to be possible to configure two devices in a primary / secondary configuration, but it is not clear how this might be represented by the API and hasn't been implemented.
+
+## Discovery
+
+Once credentials are successfully added to the bridge, any rooms (devices) detected will be added as things to the inbox.
+
+## Thing Configuration
+
+There are two types of thing, "My Warmup Account" and "Room".
+
+My Warmup Account needs configuration with your username and password for the my.warmup.com site. You can configure the refresh interval in seconds, it defaults to 300s (5 minutes).
+
+Rooms are configured automatically with a Serial Number on discovery, or can be added manually using the "Device Number" from the device, excluding the last 3 characters. The only supported temperature change is an override, through a default duration configured on the thing. This defaults to 60 minutes.
+
+## Channels
+
+
+| channel  | type   | description                  | read only |
+| --------- | -------- | ------------------------------ | ----------- |
+| currentTemperature | Number:Temperature | Currently reported temperature | true |
+| targetTemperature | Number:Temperature | Target temperature | false |
+
+## Full Example
+
+### .things file
+
+```
+Bridge warmup:my-warmup:MyWarmup [ username="test@example.com", password="test", refresh=300 ]
+{
+    room    bathroom    "Home - Bathroom"   [ serialNumber="AABBCCDDEEFF", overrideDuration=60 ]
+}
+```
+
+### .items file
+
+```
+Number:Temperature bathroom_temperature    "Temperature [%.1f °C]" <temperature> (GF_Bathroom, Temperature)    ["Temperature"] {channel="warmup:room:MyWarmup:bathroom:currentTemperature"}
+Number:Temperature bathroom_setpoint    "Set Point [%.1f °C]" <temperature> (GF_Bathroom)    ["Set Point"] {channel="warmup:room:MyWarmup:bathroom:targetTemperature"}
+```
+
+### Sitemap
+
+```
+Text label="Bathroom" icon="sofa" {
+    Text item=bathroom_temperature
+    Setpoint item=bathroom_setpoint step=0.5
+}
+```
+
+## Todo
```

Comment:
removed - is there anywhere you'd recommend to keep this? They're more ideas for if other people want to put effort into extending the binding than a todo list for me. I've focused on the currently implemented features as a complete usable MVP rather than a full client.

Reply:
You could open a GitHub issue in this repo.